output/data_plotting.py

```diff
@@ -1,48 +1,81 @@
 # Import the necessary libraries
-# import seaborn as sns
+import os
 import input.variables as var
 import numpy as np
-# import pandas as pd
 import matplotlib.pyplot as plt
 import matplotlib
 
-matplotlib.rcParams['text.usetex'] = True
+matplotlib.rcParams['text.usetex'] = True # To use latex (Needs to have miktex or another tex processor)
+matplotlib.rcParams.update({'font.size': 15})  # Size of font of plot
 
-# Load data
-phi_t_list = []
-inner_phi = np.zeros((var.D, var.D))
-first_row_phi = []
-for p in range(1, var.number_iterations + 1):
+# Change the current directory:
+# os.chdir('C:\\Users\\Elram Figueroa\\PycharmProjects\\sine-gordon\\output\\test_dx_0.5_dt_0.01_D_201_F_0.1')
+os.chdir('C:\\Users\\Elram Figueroa\\PycharmProjects\\sine-gordon\\output\\test_dx_0.5_dt_0.01_D_201_F_-0.1')
+
+# Load Data center
+# center = np.loadtxt('center.txt')  # For f>0
+center_neg = np.loadtxt('center_neg.txt') # For f<0
+
+
+# Load data soliton
+phi_t_list = []  # This list will contain all data from t_i.txt
+inner_phi = np.zeros((var.D, var.D))  # Initializing the matrix
+# first_row_phi = [] # If we need one row only of every matrix, use this
+for p in range(0, len(center_neg - 1)):
     inner_phi = np.loadtxt('t_' + str(p) + '.txt')
     phi_t_list.append(inner_phi)
-    first_row_phi.append((inner_phi[1]))
+    # first_row_phi.append((inner_phi[1]))
 
 
-
-# Constructing the axes
+# Constructing the axes soliton 2D
 # x_axes = pd.DataFrame(x_axe)
 x_axes = np.linspace(0, var.dx * var.D, num=201, axis=-1)
 X = x_axes
 Y = X
 X, Y = np.meshgrid(X, Y)
 
-
+# Plot Center
 """
-# figure
+# Create x_axis
+x_axes_center = np.linspace(0, len(center)-1)
+
+# Figure 1
 fig1 = plt.figure(1)
+plt.plot(x_axes_center, center, linewidth=2, label='$f=0.1$, $\gamma=0.1$')
+plt.plot(x_axes_center, center_neg, linewidth=2, label='$f=-0.1$, $\gamma=0.1$')
+plt.xlabel(r'$Time$', fontsize=17)
+plt.ylabel(r'$\dot{\psi}(\delta)$', fontsize=17)
+plt.xticks(fontsize=16)
+plt.yticks(fontsize=16)
+plt.legend(shadow=True, fontsize='x-large')
+plt.show()
+"""
```

Comment:
Same, build a method on order to plot, like receive some data and build the plot as you want.